### ISSUE TITLE
No way to set node name with mad

### ISSUE THREAD

**Issue by xafizoff** [synrc/mad#90]
AFAIK, it is not possible to start node with the given name. Even if I set
```erlang
-name mynode@127.0.0.1
```
in `vm.args`, I get `nonode@nohost`

**Comment by xafizoff** [comment]
Actually, `mad` can just "generate" start script, so we can start with command
```bash
$(mad start)
```
just like with `$(mad attach)`.

**Comment by xafizoff** [comment]
Solved with
```bash
$ mad bun beam myapp && tar -xzf myapp.tgz && chmod +x bin/start &&  && bin/start
```

**Comment by 5HT** [comment]
No no, this is not a solution, you're right mad should do the chmod.

**Comment by 5HT** [comment]
And mad start and mad releases are totally different things! Don't mix them in Issues.

**Comment by xafizoff** [comment]
The issue was not about start or release, it was about node name...

**Comment by 5HT** [comment]
Yeah, but setting name in release is known and trivial task by using vm.args.

**Comment by xafizoff** [comment]
I didn't know about `beam`-release... Anyways in dev-env it's not necessary.

**Comment by 5HT** [comment]
> it's not necessary

it could be. when you debug distributed setup (database e.g).
